data/org.eclipse.birt.report.data.oda.jdbc.ui/src/org/eclipse/birt/report/data/oda/jdbc/ui/provider/JdbcMetaDataProvider.java

```diff
@@ -44,6 +45,7 @@
 	private Properties props;
 	private Connection connection;
 	private boolean connect_fail = false;
+	private static final String OT_SCHEMA_PREFIX = "OTSCPF";
```

Comment:
The flag name cannot be related to the company name. It should be only neutral or feature related.

Reply:
Actually BRS has used that name.
Yes you are right we cannot have company name in OS code.